### PR TITLE
Expose -cookie-path as configuration parameter

### PR DESCRIPTION
**Pull request by costelmoraru**
## Description

Exposes the Path attribute of the cookie as config parameter `-cookie-path` to allow be more flexible cookie configuration.

## Motivation and Context
This is the PR for the issue #121 

## How Has This Been Tested?
Unit tests are passing.
Tested in our environments.

## Checklist:

- [x] My change requires a change to the documentation or CHANGELOG.
- [x] I have updated the documentation/CHANGELOG accordingly.
- [x] I have created a feature (non-master) branch for my PR.

### CODE REVIEW

**Comment by JoelSpeed** [comment]
Just one little thing before this gets merged, could you update the documentation to add this flag to the list of flags?

**Comment by costelmoraru** [comment]
Good catch, thank you for that. 
I've just pushed the updated documentation.

File: README.md [hunk 1]
```diff
@@ -261,6 +261,7 @@ Usage of oauth2_proxy:
   -client-secret string: the OAuth Client Secret
   -config string: path to config file
   -cookie-domain string: an optional cookie domain to force cookies to (ie: .yourcompany.com)
+  -cookie-path string: an optional cookie path to force cookies to (ie: .yourcompany.com/foo)
```
Comment (JoelSpeed) [inline README.md]: Should the path example include the domain? The path is just meant to be `/foo` not `.yourcompany.com/foo` when set isn't it?

Comment (costelmoraru) [reply README.md]: Correct, the path value is just `/foo` without the domain. I considered a better example if used in the context of -cookie-domain, the parameter right above, but I guess makes sense to have just the example of the path in there.
Changed it to `/foo`.

Comment (JoelSpeed) [reply README.md]: Maybe I'm being a little overly cautious, I just wondered if people might assume the value they should put is `<domain>/<uri>` based on that hint 🤷‍♂️  Thanks for updating

**Pull request merged**